package-lock.json

```diff
@@ -1387,9 +1397,9 @@
       }
     },
     "is-core-module": {
-      "version": "2.1.0",
-      "resolved": "https://registry.npmjs.org/is-core-module/-/is-core-module-2.1.0.tgz",
-      "integrity": "sha512-YcV7BgVMRFRua2FqQzKtTDMz8iCuLEyGKjr70q8Zm1yy2qKcurbFEd79PAdHV77oL3NrAaOVQIbMmiHQCHB7ZA==",
```

Comment:
I think we are supposed to delete this older version to avoid a conflict, but I'm 100% sure.

Reply:
Hi Jacob, this is a good finding, but actually this part is already deleted, github is showing that this has been updated, which is fine. If you look at the PR page, scroll down there is a section called Conflicting files, that is where you can find which files are conflicting.